doc/index.rst

```diff
@@ -370,6 +376,27 @@ For `Travis CI`, you need at least the following changes in `.travis.yml`:
 
 A successful example from `swri-robotics/mapviz <https://github.com/swri-robotics/mapviz/blob/49b0c5748950a956804e1976cfd7a224fa3f3f7d/.travis.yml>`_.
 
+Run industrial_ci on local host
+---------------------------------------
+
+Since version 0.3.3, you can run `industrial_ci` on your local host. This can be useful e.g. when you want to integrate industrial_ci into your CI server.
```

Comment:
[0.3.3 is now tagged/released](https://github.com/ros-industrial/industrial_ci/releases/tag/0.3.3) as forseen in the doc.